cases_high_level_simulation/test_3d_unit_time_dep_contact/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake) # main (top) cmake dir
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+STRING( REGEX REPLACE ".*/(.*)" "\\1" CURRENT_FOLDER ${CMAKE_CURRENT_SOURCE_DIR} )
+PROJECT("${CURRENT_FOLDER}")
+
+include(ImportSPHINXsysFromSource_for_3D_build)
+
+SET(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib)
+SET(EXECUTABLE_OUTPUT_PATH "${PROJECT_BINARY_DIR}/bin/")
+SET(BUILD_INPUT_PATH "${EXECUTABLE_OUTPUT_PATH}/input")
+SET(BUILD_RELOAD_PATH "${EXECUTABLE_OUTPUT_PATH}/reload")
+
+file(MAKE_DIRECTORY ${BUILD_INPUT_PATH})
+execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${BUILD_INPUT_PATH})
+
+set(FILES_STL			
+				"plate.stl"
+				"plate2.stl"
+				"mock_stent.stl"
+				"vessel_cylinder.stl"
+				"plate_stent.stl"
+				)
+foreach(STL_FILE ${FILES_STL})
+	file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/input/${STL_FILE} DESTINATION ${BUILD_INPUT_PATH})
+endforeach()
+
+aux_source_directory(. DIR_SRCS)
+ADD_EXECUTABLE(${PROJECT_NAME} ${DIR_SRCS})
+
+add_test(NAME ${PROJECT_NAME}_particle_relaxation 
+		 COMMAND ${PROJECT_NAME} --r=true
+		 WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+
+if(NOT SPH_ONLY_STATIC_BUILD) # usual dynamic build
+	if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
need to run the test in Windows system.

Reply:
what do you mean here?

Reply:
@Xiangyu-Hu we should make a pipeline for windows

Reply:
on: push
jobs:
  test:
    strategy:
      matrix:
        platform: [ubuntu-latest, macos-latest, windows-latest]
    runs-on: ${{ matrix.platform }}
    steps:
    - uses: actions/checkout@v1
    - uses: actions/setup-node@v1
      with:
        version: 12
    - run: npm install-ci-test
    - uses:










It looks like this:

  publish:
    needs: [build]
    steps:
    - uses: actions/checkout@v1